.github/workflows/gardening.yml

```diff
@@ -0,0 +1,36 @@
+name: Repo gardening
+
+on:
+  issues:
+    types: [opened, reopened, edited, closed]
+  issue_comment:
+    types: [created]
```

Comment:
Would it make sense to run also when an issue comment gets edited? It can happen that the issue comment adds a wrong reference, and then the comment gets updated with the correct reference, but that change isn't picked up.

Reply:
I'm afraid we can't do that. It would get us into a bit of a loop, where the bot edits its own comment to add more references, and this triggers another run of the bot, etc.

Reply:
Oh, yeah, that makes sense.